manage.py

```diff
@@ -1186,6 +1186,13 @@ def _add_athena_subparser(subparsers):
 
     athena_parser.add_argument('--table_name', help=ARGPARSE_SUPPRESS)
 
+    athena_parser.add_argument(
+        '--schema_override',
```

Comment:
Just realized you're using an underscore as a delimiter in the flag. Does that mean it has to be used as `--schem_override` in the cli, or will `--schema-override` also work? Idk if argparse enforces it...

I only mention this because our other commands/flags are delimited with a hyphen if there are two words (ie: `--test-files`, `--alarm-description`, etc...). But now that I'm looking, it seems we've been following different patterns here 🤣 . Maybe we should choose one and standardize on it.

Reply:
Merging now @ryandeivert, we'll need to revisit later